### ISSUE TITLE
Do you have char mask when the dataset is total-text?

### ISSUE THREAD

**Issue by zty8599** [MhLiao/MaskTextSpotterV3#46]
Hello,i use your model to train my own datastes , that include character-level segmentation annotations and word-level annotations for same data type. When i training datasets only include word-level annotations like total-text ,the recognition result is not good . But when two types of  annotations are uesd,the result is better. Why?
    We guess the reason is char mask,  when dataset is  synthtext,char mask is created by annotations ; when dataset is total-text,char mask is created by neural network，the accuracy of char mask  is  low, so the final  recognition  result is bad. Right?Or when dataset is total-text,there is no char mask.
     Looking forward to your reply，thanks.
